content/en/apps/concepts/prerequisites.md

```diff
@@ -6,11 +6,12 @@ description: >
   Tools and background skills that are helpful for developing CHT apps
 relatedContent: > 
   apps/tutorials/local-setup
+  core/guides/docker-setup
 ---
 There are no set prerequisites for users of CHT apps, yet the following are helpful for developing CHT applications.
 
 ## Test Instance
-To build your own application using the Core Framework you will need an instance set up for testing. You can set up a local instance by [following these instructions](https://github.com/medic/cht-core/blob/master/INSTALL.md)).
+To build your own application using the Core Framework you will need an instance set up for testing. You can set up a local instance by [following these instructions]({{< relref "apps/tutorials/local-setup" >}}).
```

Comment:
We've been using `ref` instead of `relref`, which can be [better when it comes to SEO](https://moz.com/blog/relative-vs-absolute-urls-whiteboard-friday).